content/en/docs/architecture/branching.md

```diff
@@ -10,42 +10,30 @@ The number of repositories and branches involved in assembling an OpenShift rele
 Every OCP release travels through four phases - normal development, feature freeze, code freeze, and general availability. This section details what to expect from each of these phases and the types of merge requirements that exist for PRs in each phase.
 
 ## Normal Development For 4.x Release
-During normal development, active development branches are open to all changes.
+The vast majority of the time active development branches are open to all changes.
 
 |*Branch*|*Promotes To*|*Merge Criteria*|*Fast-Forwarded?*|
 |-|-|-|-|
 |`master/main`|`ocp/4.x`|`lgtm`, `approved`|no|
-|`release-4.x-n`|`ocp/4.x-n`|`lgtm`, `approved`, `bugzilla/valid-bug`, `cherry-pick-approved`|no|
+|`release-4.x-n`|`ocp/4.x-n`|`lgtm`, `approved`, `jira/valid-bug`, `cherry-pick-approved`|no|
 |`release-4.x`|nowhere|merges blocked|yes|
 |`release-4.x+1`|`ocp/4.x+1`|merges blocked|yes|
 
-## Feature Freeze For 4.x Release
-During feature freeze, the expectation is that merges to the active development branch are fixing bugs, so we require that all PRs that land have a valid bug label.
+The primary exception to this rule is when [the Technical Release Team](../release-oversight/the-technical-release-team/) decides to enable the `/acknowledge-critical-fixes-only` label.
 
-|*Branch*|*Promotes To*|*Merge Criteria*|*Fast-Forwarded?*|
-|-|-|-|-|
-|`master/main`|`ocp/4.x`|`lgtm`, `approved`, `bugzilla/valid-bug`|no|
-|`release-4.x-n`|`ocp/4.x-n`|`lgtm`, `approved`, `bugzilla/valid-bug`, `cherry-pick-approved`|no|
-|`release-4.x`|nowhere|merges blocked|yes|
-|`release-4.x+1`|`ocp/4.x+1`|merges blocked|yes|
-
-### No-Feature-Freeze Process Pilot
-
-Some teams are participating in a pilot of a different process where Feature Freeze phase does not affect merge criteria on the `master/main` branch. Repositories that belong to these teams merge pull requests to their `master/main` branches if they are labelled with one of the following sets of labels:
-
-- `lgtm`, `approved`, `bugzilla/valid-bug` (for bugfix pull requests)
-- `lgtm`, `approved`, `px-approved`, `docs-approved`, `qe-approved` (for feature pull requests)
```

Comment:
It's  not a formal pilot anymore, since the 2024-01-31 `Removing the no-feature-freeze merge exception` email, but some repos still require these pre-merge acceptance tags on their dev branch.  For example:

```console
openshift/release $ grep -hc qe-approved core-services/prow/02_config/openshift/*/_prowconfig.yaml | sort | uniq -c
    395 0
     31 1
     49 2
```

We may want to doc various flavors of dev-branch labeling to try and consolidate around established patterns?  Or point out that the dev branch requirements are more diverse, and not really something that test-platform branching is concerned with?

Reply:
I'm going to move the legacy no-feature-freeze details to the FAQ.